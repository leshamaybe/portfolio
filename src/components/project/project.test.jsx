import Project from './Projects';
import { render } from '@testing-library/react';

describe('Project', () => {
    it('Project renders correctly', () => {
        const { container } = render(<Project />);
        expect(container).toMatchSnapshot();
    });
});
