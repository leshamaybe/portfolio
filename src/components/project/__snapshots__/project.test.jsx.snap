// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Project renders correctly 1`] = `
<div>
  <div
    class="box"
    style="transform: translateY(0%) translateZ(0);"
  >
    <div
      class="bg"
    >
      <div
        class="inner"
      >
        <a
          class="btn btn"
          href="/"
        >
          VIEW PROJECT
        </a>
        <h3 />
        <p />
      </div>
    </div>
    <h2 />
  </div>
</div>
`;
